docs/functions.json

```diff
@@ -511,21 +511,6 @@
             }
         ]
     },
-    {
-        "signature": "format_input(urn)",
```

Comment:
this function was definitely weird and annoying that it had to be the operand for all migrated rulesets on @step.value

Reply:
yes, win!